lib/src/dart_formatter.dart

```diff
@@ -114,6 +114,7 @@ class DartFormatter {
     // Parse it.
     var parser = new Parser(stringSource, errorListener);
     parser.enableOptionalNewAndConst = true;
+    parser.enableSetLiterals = true;
```

Comment:
[nit] this looks like a good place for cascades

Reply:
FWIW, I've never found the syntax to be much of an improvement, and it's annoying to have to mess with the semicolons around it. Here, it doesn't even save a line. :-/